brave/brave-ios
# Fix #2283: Added New Haptic Feedback Behavior

## Summary of Changes

*Note to Merger*:
Please add merge description:
> New feedback for:
> - Switching tabs (including swiping bottom toolbar)
> - Sponsored image logo selection
> - Selecting image credit for normal backgrounds

This pull request fixes issue #2283 


## Submitter Checklist:


## Test Plan:


- Sponsored Image:

1. Load a new tab that is a sponsored image.
1. Click on the sponsor's logo
1. Should feel haptic feedback / vibration

- Background Image Credit:

1. Load a normal background tab.
1. Click on the photographer's name.
1. Should feel haptic feedback / vibration (and see menu come up)

- Tab Switching:

> Any time a new tab is focused, you should feel the haptic feedback, some cases to test:

- [ ] Creating a new tab (this is focusing a new tab)
- [ ] Deleting a tab (this is focusing a new tab, since current one is gone)
- [ ] Deleting the _last_ tab (have one tab open, delete it via the long-press menu on tab tray)
- [ ] Swiping the bottom toolbar to navigate between tabs
- [ ] Opening the tab tray and creating a new tab
- [ ] Opening the tab tray and selecting a different tab



## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
